lib/repl.js

```diff
@@ -1067,7 +1067,7 @@ REPLServer.prototype.memory = function memory(cmd) {
           self.lines.level.push({
             line: self.lines.length - 1,
             depth: depth,
```

Comment:
It is not a guideline yet, just a comment in some PR or proposition that I recall)